helm/minio/templates/statefulset.yaml

```diff
@@ -131,8 +131,48 @@ spec:
           ports:
             - name: {{ $scheme }}
               containerPort: {{ .Values.minioAPIPort }}
+              protocol: TCP
             - name: {{ $scheme }}-console
               containerPort: {{ .Values.minioConsolePort }}
+              protocol: TCP
+          {{- if .Values.customLivenessProbe }}
+          livenessProbe: {{- include "minio.tplvalues.render" (dict "value" .Values.customLivenessProbe "context" $) | nindent 12 }}
+          {{- else if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /minio/health/live
```

Comment:
`/minio/health/live` depend on status of other applications. We can't kill minio for some applications are offline. And if minio are `busy` , we can't kill minio also that will cause the minio-cluster unavailable.  And We never kill minio.

Reply:
So I understand this /minio/health/live interface, but I can't tell whether minio is abnormal?

Reply:
Yeah. Not available for now.

Reply:
Are there other interfaces available?

Reply:
None. We never kill minio.